backend/src/Controller/Rate/RatingController.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace App\Controller\Rate;
+
+use App\AutoMapping;
+use App\Request\Rate\RatingCreateRequest;
+use App\Service\Rate\RatingService;
+use stdClass;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Serializer\SerializerInterface;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+use Nelmio\ApiDocBundle\Annotation\Security;
+use OpenApi\Annotations as OA;
+use App\Controller\BaseController;
+
+/**
+ * @Route("v1/rating/")
+ */
+class RatingController extends BaseController
+{
+    private $autoMapping;
+    private $ratingService;
+    private $validator;
+
+    public function __construct(SerializerInterface $serializer, AutoMapping $autoMapping, ValidatorInterface $validator, RatingService $ratingService)
+    {
+        parent::__construct($serializer);
+        $this->autoMapping = $autoMapping;
+        $this->validator = $validator;
+        $this->ratingService = $ratingService;
+        $this->validator = $validator;
+    }
+
+    /**
+     * store: Create rating.
+     * @Route("ratingbystore", name="createRatingByStore", methods={"POST"})
```

Comment:
this route name must change, since we need to make this API general for al cases